.gitignore

```diff
@@ -5,3 +5,4 @@ count.out
 test
 profile.out
 tmp.out
+.idea
```

Comment:
revert this.

Reply:
thanks for your review,has been reset,can you tell me why? please I'll remember (same file: .vscode)